src/esphomelib/application.h

```diff
@@ -158,8 +160,7 @@ class Application {
    * @param tx_buffer_size The size of the printf buffer.
    * @return The created and initialized LogComponent.
    */
-  LogComponent *init_log(uint32_t baud_rate = 115200,
-                         size_t tx_buffer_size = 512);
+  LogComponent *init_log(uint32_t baud_rate = 115200, size_t tx_buffer_size = 512);
```

Comment:
Please remove these cleanup changes. It makes it hard to review this PR (and makes merge conflicts that much harder to fix). If you want to fix formatting in this file, please do it in another PR.